apps/backend/src/services/integration/github.service.js

```
@@ -32,20 +32,66 @@
+    const issueOrPR = payload.issue || payload.pull_request;
+    if (!issueOrPR) {
+        console.log('No issue or pull request found in the payload.');
+        return;
+    }
+    const githubUsername = user.integration.github.userName;
+
+    // Check if the issue/PR is assigned to the user
+    const isAssignedToUser = issueOrPR.assignees.some(assignee => assignee.login === githubUsername);
+
+    // Check if the user is a reviewer for the PR
+    const isReviewer = issueOrPR.requested_reviewers && issueOrPR.requested_reviewers.some(reviewer => reviewer.login === githubUsername);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)